warning: `QString::SkipEmptyParts` is deprecated

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp: In member function ‘void OpenMS::TOPPASBase::openFilesInTOPPView(QStringList)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:1515:94: warning: ‘QString::SkipEmptyParts’ is deprecated [-Wdeprecated-declarations]
 1515 |         files = files.join("#SpLiT_sTrInG#+#SpLiT_sTrInG#").split("#SpLiT_sTrInG#", QString::SkipEmptyParts);
      |                                                                                              ^~~~~~~~~~~~~~
In file included from /usr/include/qt/QtCore/qobject.h:47,
                 from /usr/include/qt/QtWidgets/qwidget.h:45,
                 from /usr/include/qt/QtWidgets/qframe.h:44,
                 from /usr/include/qt/QtWidgets/qabstractscrollarea.h:44,
                 from /usr/include/qt/QtWidgets/qmdiarea.h:44,
                 from /usr/include/qt/QtWidgets/QMdiArea:1,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/EnhancedWorkspace.h:40,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPASBase.h:42,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp:35:
/usr/include/qt/QtCore/qstring.h:605:9: note: declared here
  605 |         SkipEmptyParts Q_DECL_ENUMERATOR_DEPRECATED
      |         ^~~~~~~~~~~~~~
```

```shell
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/LayerDataBase.cpp: In member function ‘void OpenMS::LayerDataBase::updatePeptideHitAnnotations_(OpenMS::PeptideHit&)’:
/home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/LayerDataBase.cpp:295:71: warning: ‘QString::SkipEmptyParts’ is deprecated [-Wdeprecated-declarations]
  295 |       QStringList lines = peak_anno.split(QRegExp("[\r\n]"), QString::SkipEmptyParts);
      |                                                                       ^~~~~~~~~~~~~~
In file included from /usr/include/qt/QtCore/qdatetime.h:44,
                 from /usr/include/qt/QtCore/QDate:1,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/DATASTRUCTURES/DateTime.h:40,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/DataProcessing.h:39,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/METADATA/ChromatogramSettings.h:43,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/MSChromatogram.h:38,
                 from /home/USER/Development/OpenMs/src/openms/include/OpenMS/KERNEL/MSExperiment.h:40,
                 from /home/USER/Development/OpenMs/src/openms_gui/include/OpenMS/VISUAL/LayerDataBase.h:42,
                 from /home/USER/Development/OpenMs/src/openms_gui/source/VISUAL/LayerDataBase.cpp:35:
/usr/include/qt/QtCore/qstring.h:605:9: note: declared here
  605 |         SkipEmptyParts Q_DECL_ENUMERATOR_DEPRECATED
      |         ^~~~~~~~~~~~~~
```
One more file

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -1513,7 +1513,7 @@ namespace OpenMS
       if (ret == QMessageBox::Yes)
       {
 #pragma GCC diagnostic push
-#pragma GCC diagnostic ignore "-Wdeprecated-declarations"
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
         files = files.join("#SpLiT_sTrInG#+#SpLiT_sTrInG#").split("#SpLiT_sTrInG#", QString::SkipEmptyParts);
 #pragma GCC diagnostic pop
       }

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -1512,6 +1512,10 @@ namespace OpenMS
       if (ret == QMessageBox::Cancel) return; // Escape was pressed
       if (ret == QMessageBox::Yes)
       {
+        /*
+         * Suppressed warning QSTring::SkipEmptyParts and QString::SplitBehaviour is deprecated
+         * QT::SkipEmptyParts and QT::SplitBehaviour is added or modified at Qt 5.14
+         */
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
         files = files.join("#SpLiT_sTrInG#+#SpLiT_sTrInG#").split("#SpLiT_sTrInG#", QString::SkipEmptyParts);

--- a/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
+++ b/src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
@@ -1512,7 +1512,10 @@ namespace OpenMS
       if (ret == QMessageBox::Cancel) return; // Escape was pressed
       if (ret == QMessageBox::Yes)
       {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignore "-Wdeprecated-declarations"
         files = files.join("#SpLiT_sTrInG#+#SpLiT_sTrInG#").split("#SpLiT_sTrInG#", QString::SkipEmptyParts);
+#pragma GCC diagnostic pop
       }
     }
     

